### DISCUSSION TITLE
How to initialize chakra ui with bun and its npx equvalent bunx?

### DISCUSSION THREAD

**Discussion by VladRafli** [chakra-ui/chakra-ui#9346] [Q&A] [answered]
Hello, I had problem initializing Chakra UI using bun when running installation guide on step 2:

```
npx @chakra-ui/cli snippet add
```

Trying to change npx to bunx resulting in error like this:

```
error: could not determine executable to run for package exec

error: postinstall script from "esbuild" exited with 1
```

This might be **esbuild** or **bun** issue, but there might be someone can shed some light that might know about this is much appreciated.

Thank you.

**Comment by segunadebayo** [comment, marked as answer]
That `bunx` error comes from bun/esbuild, not the Chakra CLI.

Use `npx @chakra-ui/cli snippet add` or `pnpm dlx @chakra-ui/cli snippet add`.
